Makefile

```diff
@@ -373,3 +373,17 @@ pod-failover-test-containers:
 .PHONY: upgrade-test
 upgrade-test:
 	go test -v -timeout=0 ${GOTAGS} ./test/upgrade
+
+.PHONY: install-az-analyze
+install-az-analyze:
+	go install ./cmd/az-analyze
+	mkdir -p /var/tmp/azuredisk-csi-driver/cmd/az-analyze/config
+	rm -f /var/tmp/azuredisk-csi-driver/cmd/az-analyze/config/az-analyze.yaml
+	ln -s $(PWD)/cmd/az-analyze/config/az-analyze.yaml /var/tmp/azuredisk-csi-driver/cmd/az-analyze/config/az-analyze.yaml
```

Comment:
Will go install always install the tool under the var/tmp/.. folder? Can we give any other path, possibly that uses $GOBIN? So that the driver gets installed at $GOBIN and we can also create the link to the config file at that location.

Reply:
The tool will be installed at $GOPATH/bin. Only the link to the config file is under var/tmp/.. folder.

Reply:
When we make changes in accordance to what Ed suggested below to support multiple search paths based on global, per-user and current working directory, we wouldn't need to copy this file when we run "make install-az-log". We could just mention in the README files that these config paths are supported.

Reply:
Then since we don't need to copy the file, should I remove the make command "make install-az-log"? Users can install by just running "go install".

Reply:
I think we should still keep the make command, to keep it consistent with how we build and install everything else in the driver.